code/modules/mob/living/emote.dm

```diff
@@ -503,3 +503,102 @@
 	key = "exhale"
 	key_third_person = "exhales"
 	message = "breathes out."
+
+/datum/emote/living/bababooey
+	key = "bababooey"
+	key_third_person = "bababooeys"
+	message = "spews bababooey."
+	emote_type = EMOTE_AUDIBLE
+	muzzle_ignore = FALSE
+	vary = TRUE
+
+/datum/emote/living/bababooey/can_run_emote(mob/living/user, status_check = TRUE , intentional)
+	. = ..()
+	if(. && iscarbon(user))
+		var/mob/living/carbon/C = user
+		return !C.silent
+
+/datum/emote/living/bababooey/get_sound(mob/living/user)
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(H.dna.species.id == "human" && (!H.mind || !H.mind.miming))

```

Comment:
let moths bababooey you monster

Reply:
let borgs do it too

Reply:
ath is a moth player and a borg main do not trust their words

Reply:
I just wanna bababooey, man

Reply:
Initially planned to be human and subspecies only (I'm kind of retarded so initial commit just used modified laugh emote w/out changing much to account species code), but will make it for borgs too for additional bababooey points.